src/elements/tabs/tab-group/readme.md

```diff
@@ -55,13 +72,13 @@ It's possible to set the first selected tab using the `initialSelectedIndex` pro
 
 ## Events
 
-| Name        | Type                | Description                           | Inherited From |
-| ----------- | ------------------- | ------------------------------------- | -------------- |
-| `didChange` | `CustomEvent<void>` | Emits an event on selected tab change |                |
+| Name        | Type                                           | Description                            | Inherited From |
+| ----------- | ---------------------------------------------- | -------------------------------------- | -------------- |
+| `didChange` | `CustomEvent<SbbSelectTabChangedEventDetails>` | Emits an event on selected tab change. |                |
 
 ## Slots
 
-| Name      | Description                                                                                                                                                                                                                                                                                     |
-| --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-|           | Use the unnamed slot to add html-content to the `sbb-tab-group`. Wrap the content in a `div`, a `section`, an `article` or provide a nested `sbb-tab-group`: This is correct: `<div>Some text <p>Some other text</p></div>` This is not correct: `<span>Some text</span><p>Some other text</p>` |
-| `tab-bar` | When you provide the `sbb-tab-title` tag through the unnamed slot, it will be automatically moved to this slot. You do not need to use it directly.                                                                                                                                             |
+| Name      | Description                                                                                                                                                                                                                                                                      |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|           | Use the unnamed slot to add html-content to the `sbb-tab-group`. Wrap the content in a `sbb-tab` or provide a nested `sbb-tab-group`: This is correct: `<sbb-tab>Some text <p>Some other text</p></sbb-tab>` This is not correct: `<span>Some text</span><p>Some other text</p>` |
```

Comment:
I think we don't need the explanation for incorrect content anymore, as it is guaranteed with the new tab component